modules/apps/commerce/commerce-currency-service/src/main/java/com/liferay/commerce/currency/service/impl/CommerceCurrencyLocalServiceImpl.java

```diff
@@ -344,6 +344,12 @@ public CommerceCurrency updateCommerceCurrency(
 			rate = BigDecimal.ONE;
 		}
 
+		String currencyCode = commerceCurrency.getCode();
```

Comment:
Option 1 completed in dd73e16. 

I opted to remove the existing method completely rather than overloading since "code" as a parameter should never be a valid use.